test/reporter-metrics.test.js

```diff
@@ -1,110 +1,111 @@
-'use strict';
+/* global describe, before, it */
+'use strict'
 
-const aob = require('../');
-const expect = require('chai').expect;
+const aob = require('../')
+const expect = require('chai').expect
 
-const env = process.env;
+const env = process.env
 
-const maxIsReadyToSampleWait = 60000;
+const maxIsReadyToSampleWait = 60000
 
 describe('reporter-metrics', function () {
-  const serviceKey = `${env.AO_TOKEN_PROD}:node-bindings-test`;
+  const serviceKey = `${env.AO_TOKEN_PROD}:node-bindings-test`
 
   before(function () {
-    this.timeout(maxIsReadyToSampleWait);
-    const status = aob.oboeInit({serviceKey});
+    this.timeout(maxIsReadyToSampleWait)
+    const status = aob.oboeInit({ serviceKey })
     // oboeInit can return -1 for already initialized or 0 if succeeded.
     // depending on whether this is run as part of a suite or standalone
     // either result is valid.
     if (status !== -1 && status !== 0) {
-      throw new Error('oboeInit() failed');
+      throw new Error('oboeInit() failed')
     }
 
-    const ready = aob.isReadyToSample(maxIsReadyToSampleWait);
-    const endPoint = env.APPOPTICS_COLLECTOR || 'collector.appoptics.com';
-    expect(ready).equal(1, `should be connected to ${endPoint} and ready`);
-  });
+    const ready = aob.isReadyToSample(maxIsReadyToSampleWait)
+    const endPoint = env.APPOPTICS_COLLECTOR || 'collector.appoptics.com'
+    expect(ready).equal(1, `should be connected to ${endPoint} and ready`)
+  })
 
   it('should require an array argument', function () {
     const args = [
-      {i: 'am not an array'},
+      { i: 'am not an array' },
       'nor am i',
       0,
       false,
-      Symbol('xyzzy'),
-    ];
+      Symbol('xyzzy')
```

Comment:
In my opinion, this is not a good setting for your auto-formatting. I actually prefer lists like that being ended with a `,` if it is allowed by the language syntax. The reason for this is, that once you add something to the list, you will end up with a more straightforward change/ diff as only the line you added will appear in the diff, i.e. instead of having a diff like
```
-    Symbol('xyzzy'),
+    NewSymbol('xyzzy')
]
```
you will just get a diff like
```
+    NewSymbol('xyzzy'),
]
```
if a list always ends with a `,`.

Reply:
Yep. I also personally tend to be "pro dangle". For that reason, but also for the editing/commenting-out reason (see more here: https://eslint.org/docs/rules/comma-dangle). 

That being said, preferring to choose a popular style guide from the four offered by eslint - @standard is kind of where the zeitgeist is (@airbnb? @google? naaa....) and it is a "no dangle" style guide.

@feross has been ahead of the game with this and has done a great job [marketing](https://github.com/standard/standard#but-this-isnt-a-real-web-standard) it, so it is very popular. Since I agree with the [principle](https://github.com/standard/standard#i-disagree-with-rule-x-can-you-change-it) - there ill be "no dangle" for us (for now).